<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaPrincipal</class>
 <widget class="QMainWindow" name="JanelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JanelaPrincipal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="botaoPrimeiraPagina">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Recursos.qrc">
          <normaloff>:/Recursos/icones/arrow-left-double.png</normaloff>:/Recursos/icones/arrow-left-double.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="botaoPaginaAnterior">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Recursos.qrc">
          <normaloff>:/Recursos/icones/arrow-left.png</normaloff>:/Recursos/icones/arrow-left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPaginas">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="botaoProximaPagina">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Recursos.qrc">
          <normaloff>:/Recursos/icones/arrow-right.png</normaloff>:/Recursos/icones/arrow-right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="botaoUltimaPagina">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Recursos.qrc">
          <normaloff>:/Recursos/icones/arrow-right-double.png</normaloff>:/Recursos/icones/arrow-right-double.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDocumentos">
    <property name="title">
     <string>&amp;Documentos</string>
    </property>
    <addaction name="actionAdicionarDocumento"/>
    <addaction name="actionEditarDocumento"/>
    <addaction name="actionExcluirDocumento"/>
   </widget>
   <widget class="QMenu" name="menuTabela">
    <property name="title">
     <string>&amp;Tabela</string>
    </property>
    <addaction name="actionAtualizar"/>
    <addaction name="actionFiltrar"/>
    <addaction name="actionIrParaPagina"/>
    <addaction name="actionItensPorPagina"/>
    <addaction name="actionLimparFiltros"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>&amp;Sobre</string>
    </property>
    <addaction name="actionSobreEstePrograma"/>
    <addaction name="actionSobreQt"/>
   </widget>
   <addaction name="menuDocumentos"/>
   <addaction name="menuTabela"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdicionarDocumento"/>
   <addaction name="actionEditarDocumento"/>
   <addaction name="actionExcluirDocumento"/>
   <addaction name="separator"/>
   <addaction name="actionAtualizar"/>
   <addaction name="actionFiltrar"/>
   <addaction name="actionLimparFiltros"/>
   <addaction name="actionIrParaPagina"/>
   <addaction name="separator"/>
   <addaction name="actionSobreQt"/>
   <addaction name="actionSobreEstePrograma"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdicionarDocumento">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/icones/list-add.png</normaloff>:/Recursos/icones/list-add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Adicionar Documento</string>
   </property>
   <property name="toolTip">
    <string>Adiciona um novo documento na coleção.</string>
   </property>
   <property name="statusTip">
    <string>Adiciona um novo documento na coleção.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEditarDocumento">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/icones/edit-rename.png</normaloff>:/Recursos/icones/edit-rename.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Editar Documento</string>
   </property>
   <property name="toolTip">
    <string>Edita as informações de um  documento registrado anteriormente.</string>
   </property>
   <property name="statusTip">
    <string>Edita as informações de um documento registrado anteriormente.</string>
   </property>
  </action>
  <action name="actionExcluirDocumento">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/icones/edit-delete.png</normaloff>:/Recursos/icones/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Excluir Documento</string>
   </property>
   <property name="toolTip">
    <string>Exclui o documento selecionado na lista</string>
   </property>
   <property name="statusTip">
    <string>Exclui o documento selecionado na lista</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAtualizar">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/icones/view-refresh.png</normaloff>:/Recursos/icones/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Atualizar</string>
   </property>
   <property name="toolTip">
    <string>Atualiza a lista de registros de documentos.</string>
   </property>
   <property name="statusTip">
    <string>Atualiza a lista de registros de documentos.</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionFiltrar">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/icones/view-filter.png</normaloff>:/Recursos/icones/view-filter.png</iconset>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
   <property name="toolTip">
    <string>Adiciona filtros de pesquisa</string>
   </property>
   <property name="statusTip">
    <string>Adiciona filtros de pesquisa.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionIrParaPagina">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/icones/go-jump.png</normaloff>:/Recursos/icones/go-jump.png</iconset>
   </property>
   <property name="text">
    <string>Ir Para a Página</string>
   </property>
   <property name="toolTip">
    <string>Informe o número da página de registros que deseja acessar.</string>
   </property>
   <property name="statusTip">
    <string>Informe o número da página de registros que deseja acessar.</string>
   </property>
  </action>
  <action name="actionLimparFiltros">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/icones/run-build-clean.png</normaloff>:/Recursos/icones/run-build-clean.png</iconset>
   </property>
   <property name="text">
    <string>Limpar Filtros</string>
   </property>
   <property name="statusTip">
    <string>Elimina os filtros atuais.</string>
   </property>
  </action>
  <action name="actionSobreQt">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/icones/icon_Qt_60x60px.png</normaloff>:/Recursos/icones/icon_Qt_60x60px.png</iconset>
   </property>
   <property name="text">
    <string>Sobre Qt</string>
   </property>
   <property name="toolTip">
    <string>Informações sobre a versão do framework Qt utilizada.</string>
   </property>
   <property name="statusTip">
    <string>Informações sobre a versão do framework Qt utilizada.</string>
   </property>
  </action>
  <action name="actionSobreEstePrograma">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/icones/help-about.png</normaloff>:/Recursos/icones/help-about.png</iconset>
   </property>
   <property name="text">
    <string>Sobre Este Programa</string>
   </property>
   <property name="toolTip">
    <string>Informações sobre este programa.</string>
   </property>
   <property name="statusTip">
    <string>Informações sobre este programa.</string>
   </property>
  </action>
  <action name="actionItensPorPagina">
   <property name="text">
    <string>Itens por página</string>
   </property>
   <property name="toolTip">
    <string>Defina a quantidade máxima de itens mostrados por página.</string>
   </property>
   <property name="statusTip">
    <string>Define a quantidade máxima de itens exibidos por página.</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
